src/fsharp/pars.fsy

```diff
@@ -3740,9 +3741,6 @@ atomicExprQualification:
             mlCompatWarning (FSComp.SR.parsParenFormIsForML()) (lhs parseState) 
             mkSynDotParenGet lhsm dotm e $2) }
 
-  |   LBRACK  typedSeqExpr RBRACK
```

Comment:
I remember we looked at this before. But, I think this removal is affecting the error messages on the indexer and am wondering if there are any other ramifications.

Reply:
There was a problem remove this:
```fsharp
let a = [| 1 |]
let f y = a.[y:int]
```
This fails to compile.